{"uid":"81196ac222fa7a7e","name":"test_edit_contact_empty_data","fullName":"test.Contactos.test_edit_contact#test_edit_contact_empty_data","historyId":"e218ad908714a2195329772962ced733","time":{"start":1726764906798,"stop":1726764906799,"duration":1},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nTypeError: PayloadContact.build_payload_add_contact() missing 3 required positional arguments: 'firstName', 'lastName', and 'name'","statusTrace":"setup_edit_contact = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, [{'accountId': None, 'accountIsInactive': None, 'accountName': None, 'accountType': None, ...}])\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_edit_contact_empty_data(setup_edit_contact):\n        headers, contact = setup_edit_contact\n>       payload = PayloadContact().build_payload_add_contact(salutationName=\"Ms.\",\n                                                             addressPostalCode=\"0000\", addressStreet=\"none\",\n                                                             addressState=\"none\", addressCity=\"none\", addressCountry=\"none\",)\nE       TypeError: PayloadContact.build_payload_add_contact() missing 3 required positional arguments: 'firstName', 'lastName', and 'name'\n\ntest/Contactos/test_edit_contact.py:61: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726764900558,"stop":1726764900558,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_edit_contact","time":{"start":1726764906628,"stop":1726764906798,"duration":170},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_edit_contact::0","time":{"start":1726764906802,"stop":1726764906802,"duration":0},"status":"broken","statusMessage":"TypeError: list indices must be integers or slices, not str\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/espo-automation-final/espo-automation-final/test/Contactos/conftest.py\", line 61, in setup_edit_contact\n    ContactCall().delete(headers, contact['id'])\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Contactos"},{"name":"suite","value":"test_edit_contact"},{"name":"host","value":"fv-az1020-822"},{"name":"thread","value":"2107-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Contactos.test_edit_contact"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"81196ac222fa7a7e.json","parameterValues":[]}